server/neptune/workflows/activities/github.go

```diff
@@ -31,13 +31,14 @@ var HashiGetter = func(ctx context.Context, dst, src string) error {
 // wraps hashicorp's go getter to allow for testing
 type gogetter func(ctx context.Context, dst, src string) error
 
-type githubClient interface {
+type githubClient interface { //nolint:interfacebloat
```

Comment:
We can look into potentially breaking down this interface in the future. Don't want to couple that with this PR.